src/families/cardano/js-synchronisation.ts

```diff
@@ -0,0 +1,100 @@
+import type { Account, Address } from "../../types";
+import { GetAccountShape, makeScanAccounts } from "../../bridge/jsHelpers";
+import { makeSync } from "../../bridge/jsHelpers";
+import { encodeAccountId } from "../../account";
+
+import BigNumber from "bignumber.js";
+import Ada from "@cardano-foundation/ledgerjs-hw-app-cardano";
+import { str_to_path } from "@cardano-foundation/ledgerjs-hw-app-cardano/dist/utils";
+import { getCardanoResourseForAccount } from "./api/getAccount";
+import { getOperations } from "./api";
+import { getBaseAddress, getBipPathString } from "./logic";
+
+const postSync = (initial: Account, parent: Account) => parent;
```

Comment:
This is actually unneeded since it's already provided as a default value [here](https://github.com/LedgerHQ/ledger-live-common/blob/5fa6a917da4a4789d757ae67eb0679f54ce6cd52/src/bridge/jsHelpers.ts).
(It's on us to update the doc and cleanup other integrations)